test/recipes/70-test_quic_multistream.t

```diff
@@ -6,16 +6,40 @@
 # in the file LICENSE in the source distribution or at
 # https://www.openssl.org/source/license.html
 
-use OpenSSL::Test qw/:DEFAULT srctop_file/;
+use OpenSSL::Test qw/:DEFAULT srctop_file result_dir data_file/;
 use OpenSSL::Test::Utils;
+use File::Temp qw(tempfile);
+use File::Path 2.00 qw(rmtree);
 
 setup("test_quic_multistream");
 
 plan skip_all => "QUIC protocol is not supported by this OpenSSL build"
     if disabled('quic');
 
-plan tests => 1;
+plan tests => 2;
+
+if (!disabled('qlog') && $ENV{RUN_CI_TESTS} == "1") {
```

Comment:
Is this a github environment variable? I wonder if it will catch anyone out running OpenSSL tests in their own CI?

Reply:
I'll make it OSSL_RUN_CI_TESTS